### ISSUE TITLE
Remove focus ring/outline around drawer button

### ISSUE THREAD

**Issue by leifoolsen** [google/material-design-lite#4290]
MDL version: 1.1.3
Browser: Chrome
Browser version: Latest
Operating system: OSX
Operating system version: Latest

What steps will reproduce the problem:
1. Click on drawer button

What is the expected result?
Focus ring should not be visible

What happens instead of that?
Focus ring is visible

Please provide any other information below, and attach a screenshot if possible.
![screen shot 2016-04-20 at 22 49 11](https://cloud.githubusercontent.com/assets/10088486/14690003/32d554b0-074a-11e6-8977-6f95e515942f.png)

CSS to remove the "annoying" focus ring could be someting like this:

```css
.mdl-layout__drawer-button {

  outline: none;

  // Mozilla, remove focusring
  &:-moz-focusring {
    color: transparent;
    text-shadow: 0 0 0 #000;
  }
}
```

**Comment by Garbee** [comment]
The focus ring is for accessibility and should be visible until we have clear and direct guidance from the MD team on how this should be done otherwise.

**Comment by leifoolsen** [comment]
None of the other MDL buttons,https://getmdl.io/components/index.html#buttons-section, has a focus ring - so then there is an inconsistency. 

BTW:  it may look like that MDL buttons do not have a natural tab order.

**Comment by leifoolsen** [comment]
... if I click on a button, the the TAB-key should move focus to the next button.

**Comment by Download** [comment]
Normal buttons get an elevation when you tab to them; they get a bigger shadow that suggests they have moved up towards the viewer. I think this is the same effect used when the button is clicked. Maybe the default focus effect on the drawer button should mimick that? The current one looks really ugly indeed.

**Comment by Garbee** [comment]
This is superseded by issue #4480. We are going to work with the MD team while building the V2 component to get active states on the web addressed better.

Thank you.
